feat(HTTP): Refresh 5XX response pages (#35361)
mdn/content: files/en-us/web/http/status/501/index.md

{{HTTPSidebar}}

> The HTTP **`501 Not Implemented`** [server error response](/en-US/docs/Web/HTTP/Status#server_error_responses) status code means that the server does not support the functionality required to fulfill the request.

Reviewer: ```suggestion
The HTTP **`501 Not Implemented`** [server error response](/en-US/docs/Web/HTTP/Status#server_error_responses) status code means that the server does not support the functionality required to fulfill the request.

```

> This status may also include a {{HTTPHeader("Retry-After")}} header, telling the client when to retry the request to check if the functionality is supported by then.

Reviewer: ```suggestion
A response with this status may also include a {{HTTPHeader("Retry-After")}} header, telling the client that they can retry the request after some time has elapsed.
```

The HTTP **`501 Not Implemented`** [server error response](/en-US/docs/Web/HTTP/Status#server_error_responses) status code means that the server does not support the functionality required to fulfill the request.
This status may also include a {{HTTPHeader("Retry-After")}} header, telling the client when to retry the request to check if the functionality is supported by then.
A 501 response is cacheable by default unless caching headers instruct otherwise.

`501` is the appropriate response when the server does not recognize the request method and is incapable of supporting it for any resource.
If the server _does_ recognize the method, but intentionally does not support it, the appropriate response is {{HTTPStatus(405, "405 Method Not Allowed")}}.
Servers are required to support {{HTTPMethod("GET")}} and {{HTTPMethod("HEAD")}}, and therefore must not return `501` in response to requests with these methods.

> [!NOTE]
>

> > - A 501 error is not something you can fix, but requires a fix by the web server you are trying to access.

Reviewer: Is this point relevant still? (i.e.  or is the assumption that all 5xx issues are unfixable by the user?)

Author: We can add something similar to the others, edit incoming
